common/persistence/client/health_request_rate_limiter.go

```diff
@@ -63,25 +62,6 @@ type (
 		metricsHandler metrics.Handler
 		logger         log.Logger
 	}
-
-	dynamicRateLimitingOptions struct {
-		Enabled bool
-
-		RefreshInterval string // string returned by json.Unmarshal will be parsed into a duration
-
-		// thresholds which should trigger backoff if exceeded
-		LatencyThreshold float64
-		ErrorThreshold   float64
-
-		// if either threshold is exceeded, the current rate multiplier will be reduced by this amount
-		RateBackoffStepSize float64
-		// when the system is healthy and current rate < max rate, the current rate multiplier will be
-		// increased by this amount
-		RateIncreaseStepSize float64
```

Comment:
Did you move these comments somewhere? I saw there are already comments above, are those enough?

Reply:
the others pretty much cover all these, but I can combine them a little.

I also added the field at least to the docstrings. there are four of them so I didn't want to copy the whole paragraph but maybe I should. let's see what it looks like after the doc generation stuff.